bindings/nodejs/tests/getresult.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the Soletta Project
+ *
+ * Copyright (C) 2016 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var fs = require( "fs" );
+
+var jsonFilename = require( "path" ).resolve( __dirname, "results.json" );
+
+var date = new Date(),
+  data = "",
+  caseList = [],
+  setList = [],
+  allList = [],
+  test = {},
+  testInfo = {};
+
+exports.getTestResult = function( status, success, failure  ) {
+
+  if ( status.result ) {
+    success = "PASS";
+  } else if ( !status.result ) {
+    failure = "FAIL";
+  }
+
+  testInfo = {
+    "message": (  status.runtime  + ": " + status.message ),
+    "result": ( status.result ? success : failure ),
```

Comment:
Maybe all that you wanted here was something like:
```js
"result": ( status.result ? "PASS" : "FAIL" ),
```
With no need for `success` and `failure` parameters?